### PR TITLE
production/GFS.v16: gfsv16 physics updates

### PR DESCRIPTION
**Pull request by junwang-noaa**
update gfsphysics update, postlib and parallel netcdf on hera

### CODE REVIEW

File: modulefiles/hera.intel/fv3 [hunk 1]
```diff
@@ -41,16 +41,19 @@ module load jasper/1.900.1
...
 ##
 ## load ESMF library for above compiler / MPI combination
 ## use pre-compiled EMSF library for above compiler / MPI combination
 ##
-module use -a /scratch1/NCEPDEV/nems/emc.nemspara/soft/modulefiles
-module load hdf5_parallel/1.10.6
-module load netcdf_parallel/4.7.4
-module load esmf/8.0.1bs08
+#module use -a /scratch1/NCEPDEV/nems/emc.nemspara/soft/modulefiles
+#module load hdf5_parallel/1.10.6
```
Comment (yangfanglin) [inline modulefiles/hera.intel/fv3]: still need HDF module

**Review by yangfanglin — COMMENTED** [review]
need to add hd-parallel module to Hera.intel/fv3

**Review by WenMeng-NOAA — APPROVED** [review]
post part looks good for me.

**Comment by junwang-noaa** [comment]
Fanglin,

Code is updated, would you please review again? Thanks

**Review by yangfanglin — COMMENTED** [review]
The changes look fine.

Please hold on creating a tag.  There might be an extra change in physics. I am working with the developer to review it.

**Review by yangfanglin — APPROVED** [review]
The changes look fine for me.  Need to update to official  netcdf and esmf modules when they become available

**Pull request merged**